mvg/mvg.py

```diff
@@ -765,6 +765,111 @@ def get_analysis_results(self, request_id: str) -> dict:
 
         return response.json()
 
+    # Labels
+    def create_label(
+        self,
+        sid: str,
+        timestamp: int,
+        label: str,
+        severity: int,
+        notes: Optional[str] = "",
+    ):
+        """Create a label for a measurement
+
+        Parameters
+        ----------
+        sid : str
+            Id of the source for the measurement
+        timestamp : int
+            Timestamp of the measurement to label
+        label : str
+            A string label to attach to the measurement
+        severity : int
+            Severity of the label as a positive integer
+        notes : Optional[str], optional
+            Optional notes for the label, by default ""
+        """
+        logger.info("endpoint %s", self.endpoint)
+        logger.info(f"Creating label for {sid} - {timestamp}")
+
+        label_data = {"label": label, "severity": severity, "notes": notes}
+
+        self._request("post", f"/sources/{sid}/labels/{timestamp}", json=label_data)
+
+    def get_label(self, sid: str, timestamp: int) -> dict:
```

Comment:
This is not according to the API spec
- takes a list of timestamps
- (as what I can see), the timestamp argument is required, which is not according to specification